core/src/tangram.h

```diff
@@ -154,6 +156,7 @@ bool getDebugFlag(DebugFlags _flag);
 void toggleDebugFlag(DebugFlags _flag);
 
 void runOnMainLoop(std::function<void()> _task);
+void runAsyncTask(std::function<void()> _task);
```

Comment:
Could probably add some comments on how this differentiate from `runOnMainLoop`.